src/test/Common.hs

```diff
@@ -46,6 +52,24 @@ testConfig = defaultConfig & sConf . quiet .~ True
                            & cConf . testLimit .~ 10000
                            & cConf . shrinkLimit .~ 4000
 
+type SolcVersion = Version
+type SolcVersionComp = Version -> Bool
+
+solcV :: Int -> Int -> Int -> SolcVersion
```

Comment:
wouldn't simply tuple with three integers work for our use case?

Reply:
I used like that to take the smallest amount of space when writting new tests (`solcV a b c` instead of `solcV (a,b,c)`)

Reply:
I mean instead of semver, not a big deal though

Reply:
i think tuple makes more sense here

we shouldn't be optimizing for code space

Reply:
Fair enough. Fixed.